Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate.py

```diff
@@ -1797,6 +1793,24 @@ def get_url_timeline(url):
 
 LOG('command is %s' % (demisto.command(),))
 try:
+
+    params = demisto.params()
+
+    api_token = params['APIToken']
+    base_url = params['baseURL']
+    use_ssl = not params.get('insecure', False)
+    malicious_threshold = int(params.get('dboscore_threshold', -100))
```

Comment:
you are not moving these params. did you test that the `malicious_threshold`, e.g, works as expected?